### DISCUSSION TITLE
segmentation_models_pytorch model cant travel to utils folder

### DISCUSSION THREAD

**Discussion by supratimpaul13** [qubvel/segmentation_models.pytorch#796] [Q&A]
#782 This is the code in example folder for car segmentations.

loss = smp.utils.losses.DiceLoss()
metrics = [
    smp.utils.metrics.IoU(threshold=0.5),
]

optimizer = torch.optim.Adam([ 
    dict(params=model.parameters(), lr=0.0001),
])

and it is giving me error:
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[92], line 4
      1 # Dice/F1 score - https://en.wikipedia.org/wiki/S%C3%B8rensen%E2%80%93Dice_coefficient
      2 # IoU/Jaccard score - https://en.wikipedia.org/wiki/Jaccard_index
----> 4 import segmentation_models_pytorch.utils.losses as smp_losses
      5 import segmentation_models_pytorch.utils.metrics as smp_metrics
      7 loss = smp_losses.DiceLoss()

ModuleNotFoundError: No module named 'segmentation_models_pytorch.utils'

I have installed all the necessary libraries required but cant find any solution

**Comment by Prathamdesai-07** [comment]
I've faced the same error too, it got resolve by just simply importing the 'utils' from 'segmentation_models_pytorch'. however its other varients are depreciated for counting the loss.
Here is the code snippet. 

<img width="461" alt="Screenshot 2024-02-15 011035" src="https://github.com/qubvel/segmentation_models.pytorch/assets/112389271/f7dad0d8-ed00-4edc-97e8-c74929555384">

i hope this helps.
